components/Modal.js

```diff
@@ -0,0 +1,84 @@
+/*
+  Recommended use:
+  - apply `margin-top: 0` to the first element in the modal to make it line up with the close button
```

Comment:
could we add a hack where the close button has a negative margin? Not sure if that would work

Reply:
Unless I'm misunderstanding negative margin would put it up higher in the component. I could see us adding additional margin to make it lower to account for the default margins that come with `header` elements, but it's also hard to account for the varying margins for different headings. Some elements also wouldn't have built-in margins (eg. images)

Do you have a code example for this?